backend/views/profile.js

```diff
@@ -101,81 +101,36 @@ exports.deletePhoto = function(profilePhoto) {
 
  
 
-  // PUT
-   var editProfile = function (req, res, next) {
+/**
```

Comment:
@param instead of @ params
mention param after description
sorry i told u it is params before